a3p-integration/proposals/a:upgrade-next/upgradeVaults.test.js

```diff
@@ -71,7 +54,7 @@ const tryPushPrices = async t => {
   const osmoOut = await getPriceQuote('stOSMO');
   t.is(osmoOut, '+11500000');
   const stkAtomOut = await getPriceQuote('stkATOM');
-  t.is(stkAtomOut, '+11600000');
+  await t.is(stkAtomOut, '+11600000');
```

Comment:
Noop, remove

Reply:
This is straight cherry-pick of https://github.com/Agoric/agoric-sdk/pull/9536, arguably I should have reviewed that better